dotcom-rendering/src/web/components/StickyVideo.importable.tsx

```diff
@@ -91,27 +92,70 @@ const stickyContainerStyles = (height: number) => css`
 interface Props {
 	isPlaying: boolean;
 	children: React.ReactNode;
+	id: string;
 }
 
-export const StickyVideo = ({ isPlaying, children }: Props) => {
-	const [isSticky, setIsSticky] = useState(false);
+interface StickyState {
+	isSticky: boolean;
+	isClosed: boolean;
+}
+
+export const StickyVideo = ({ isPlaying, children, id }: Props) => {
+	const [stickyState, setStickyState] = useState<StickyState>({
+		isSticky: false,
+		isClosed: false,
+	});
```

Comment:
Thanks for pointing it out. 
I was very unsure about what's the best way to do this. At first I was having some issues with the state changes and thought is there are 2 states that change, that would cause 2 different re-rendering of the component. But now that I tested again, I realised that was not the case. 

Changed the states to be separate. What do you think now?